verification/cbmc/proofs/aws_ring_buffer_init/aws_ring_buffer_init_harness.c

```diff
@@ -7,16 +7,21 @@
 #include <proof_helpers/make_common_data_structures.h>
 
 void aws_ring_buffer_init_harness() {
-    /* parameters */
-    struct aws_ring_buffer ring_buf;
+    /* Non-deterministic parameters. */
+    struct aws_ring_buffer *ring_buf = malloc(sizeof(*ring_buf));
```

Comment:
Can we create a GitHub issue to review all stack allocations in proofs and replace it with dynamic allocation to avoid implicit assumptions about object nullness?

Reply:
https://github.com/awslabs/aws-c-common/issues/803 Created one